Library/Homebrew/utils/curl.rb

```diff
@@ -75,9 +76,11 @@ def curl_with_workarounds(
         verbose:      verbose,
       }.compact
 
+      curl = use_brewed_curl ? Formula["curl"].opt_bin/"curl" : curl_executable
```

Comment:
Shouldn't this logic be in the `curl_executable` method?

Reply:
Done